### PR TITLE
Mobile version continuation

### PR DESCRIPTION
**Pull request by Thi-Ponce**
### Mobile version project
In this part of the project, I added more pages to my future portfolio following the guidelines Microverse sets for us. Those additions include:
- Cards about my future projects;
- An "About Me" section.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -77,3 +79,178 @@ ul {
 li {
   padding-right: 20px;
 }
+
+.card {
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  padding: 20px 0;
+}
```
Comment (mike2611) [inline style.css:88]: - [ ] Please make sure to follow the Figma template as close as possible, your margins are different from the ones in the Figma template

## Figma
image

## Your Project
image

File: index.html [hunk 2]
```diff
@@ -20,24 +20,128 @@
...
+        <a href="#"><i class="fab fa-medium-m fa-2x"></i></a>
+      </li>
+      <li>
+        <a href="#"><i class="fab fa-github fa-2x"></i></a>
+      </li>
+      <li>
+        <a href="#"><i class="fab fa-angellist fa-2x"></i></a>
+      </li>
+    </ul><br>
+    <button type="button" class="resume">Get my resume</button><br>
```
Comment (mike2611) [inline index.html:136]: - [ ] Please make sure to follow the Figma template as close as possible, your button is different than the one in the Figma template. :computer: 
## Figma
image
## Your Project
image

**Review by mike2611 — CHANGES_REQUESTED** [review]
Hi @Thi-Ponce , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by lily-coder — APPROVED** [review]
Hi @Thi-Ponce , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image



- [x] You can also consider writing your commit messages in the imperative tense next time instead of the past tense. To learn more on how you can make your work more professional, including commit messages, you can follow [This Link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**